cleanup.sh

```diff
@@ -406,19 +406,19 @@ for CRD in $(kubectl get crd -o name | grep logging\.banzaicloud\.io); do
 done
```

Comment:
Should we skip deleting any CRDs?

Since we have control over the job, maybe we can add a hook to make the CRD cleanup optional as a follow-up task.

Reply:
nm, I think that will have to be part of the "disable Rancher" epic when it comes.